airbyte-integrations/connectors/source-mailersend/source_mailersend/mailersend.yaml

```diff
@@ -0,0 +1,50 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["data"]
+  requester:
+    url_base: "https://api.mailersend.com/v1"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+    request_options_provider:
+      request_parameters:
+        limit: "100"
+        date_from: "{{ config['date_from'] }}"
+        date_to: "{{ config['date_to'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: "DefaultPaginator"
+      page_size_option:
+        inject_into: "request_parameter"
+        field_name: "page_size"
+      pagination_strategy:
+        type: "PageIncrement"
+        page_size: 100
+      page_token_option:
+        inject_into: "request_parameter"
+        field_name: "page"
+      url_base: "https://api.mailersend.com/v1"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  activity_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "activity"
+      primary_key: "id"
+      path: "/activity/{{ config['domain_id'] }}"
+
+streams:
+  - "*ref(definitions.activity_stream)"
```

Comment:
are you planning to add others streams?

Reply:
@marcosmarxm, not for now. This is the most useful stream. It is all events related to emails. If it was delivered, opened, bounce...